src/assets/css/main.css

```diff
@@ -200,6 +196,7 @@ blockquote {
   min-height: 60px;
   max-height: 80px;
   width: 100%;
+  height: 100%;
```

Comment:
I’m fairly certain this will lead to aspect ratio issues.

Reply:
the height 100% affects only .thumbnail with max-height limiting it to 80px. If you toggle it in the dev console you'll see how it makes all thumbnails more uniform in size